_templates/layout.html

```diff
@@ -0,0 +1,13 @@
+{% extends '!layout.html' %}
+
+{% block footer %}
+<div class="container-xl">
+    <ul class="navbar-nav">
+        <li class="nav-end-item">
+            <a class="nav-link nav-external" href="#">TODO: add more links</a>
```

Comment:
I think we can add a link to the generated `CONTRIBUTING.md` here, and Cite and Contact as in the user guide. Or other links? WDYT?